src/core/execute.c

```diff
@@ -4909,6 +4910,17 @@ static int exec_child(
                         return r;
         }
 
+        if (context->memory_ksm) {
+                if (prctl(PR_SET_MEMORY_MERGE, 1) < 0) {
```

Comment:
should be:

```c
if (prctl(PR_SET_MEMORY_MERGE, context->memory_ksm) < 0) {
```

(under the assumption the parameter to the prctl indicates whether to enable or disable the feature?)

Reply:
Done.